core/templates/dev/head/collection_editor/collection_editor.html

```diff
@@ -98,5 +158,13 @@
     {{ include_js_file('domain/editor/ChangeObjectFactory.js') }}
     {{ include_js_file('domain/editor/UndoRedoService.js') }}
     {{ include_js_file('domain/summaries/ExplorationSummaryBackendApiService.js') }}
+    {{ include_js_file('domain/utilities/UrlInterpolationService.js') }}
+
+    {{ include_js_file('exploration_editor/EditorServices.js') }}
+    {{ include_js_file('exploration_editor/ExplorationEditor.js') }}
+    {{ include_js_file('exploration_editor/GadgetValidationService.js') }}
```

Comment:
Ah yes, that's pretty ick. Maybe add a comment here too, with the unwanted imports in a block, so that they can be cleaned away easily when the time is right.

Why ExplorationEditor.js though? That file contains only controllers IIRC.